src/components/timeout-dialog/timeout-dialog.test.js

```diff
@@ -336,6 +336,35 @@ describe('/components/timeout-dialog', () => {
     });
   });
 
+  describe('display the sign out link', () => {
+    it('should not show sign out button when data-hide-sign-out-link parameter is set', () => {
+      setupDialog({
+        'data-title': 'my custom TITLE',
+        'data-message': 'MY custom message',
+        'data-message-suffix': 'My message suffix.',
+        'data-keep-alive-button-text': 'KEEP alive',
+        'data-hide-sign-out-link': true,
+      });
+
+      pretendSecondsHavePassed(780);
+
+      expect(testScope.latestDialog$element.querySelector('a#hmrc-timeout-sign-out-link')).toBeNull();
+    });
+
+    it('should show sign out button when disable-sign-out-button parameter is unset', () => {
```

Comment:
maybe make it clear that this is the default?
```suggestion
    it('should show sign out button by default', () => {
```